tests/sizeof_tcb/Makefile

```diff
@@ -1,4 +1,7 @@
 APPLICATION = sizeof_tcb
 include ../Makefile.tests_common
 
+# optionally disable messaging
+#DISABLE_MODULE += msg
```

Comment:
Wouldn't it make sense to comment this in here?

Maybe you could add this snippet also to `dist/Makefile`?